### PR TITLE
HAWQ-1395. PXF build script to support passing additional Gradle options.

### PR DESCRIPTION
**Pull request by sansanichfb**

### CODE REVIEW

File: pxf/Makefile [hunk 1]
```diff
@@ -42,6 +42,10 @@ ifneq "$(PXF_VERSION)" ""
     BUILD_PARAMS+= -Pversion="$(PXF_VERSION)"
 endif
 
+ifneq "$(GRADLEW_OPTIONS)" ""
```
Comment (shivzone) [inline pxf/Makefile]: The purpose of Makefile is to abstract the user from gradle options/commands. Should we instead call this BUILD_OPTIONS ?

Comment (sansanichfb) [inline pxf/Makefile]: Another option - add EXCLUDE_PROJECTS, which will take comma-separated list of projects.

Comment (shivzone) [inline pxf/Makefile]: Yes, that would be good

**Comment by sansanichfb** [comment]
This specific change just excludes buildRpm task, all other tasks still remain.

**Pull request closed without merging**